spring-ai-docs/src/main/antora/modules/ROOT/nav.adoc

```diff
@@ -1,10 +1,11 @@
 * xref:index.adoc[Overview]
-* xref:concepts.adoc[AI Concepts]
+** xref:concepts.adoc[AI Concepts]
 * xref:getting-started.adoc[Getting Started]
-* xref:api/index.adoc[]
-** xref:api/chatclient.adoc[]
-*** xref:api/advisors.adoc[Advisors]
-** xref:api/chatmodel.adoc[]
+* xref:api/chatclient.adoc[]
+** xref:api/advisors.adoc[Advisors]
+* xref:api/index.adoc[AI Models]
+** xref:api/chatmodel.adoc[Chat Models]
+*** xref:api/prompt.adoc[]
```

Comment:
I think the prompt stuff should move elsewhere.  All the "API" subsections are specific to a model vendor whereas this is a higher level spring ai api topic.